snap/seed_yaml.go

```diff
@@ -89,6 +91,16 @@ func ReadSeedYaml(fn string) (*Seed, error) {
 		if strings.Contains(sn.File, "/") {
 			return nil, fmt.Errorf("%s: %q must be a filename, not a path", errPrefix, sn.File)
 		}
+
+		// make sure names and file names are unique
```

Comment:
Should we make sure snap IDs explicitly listed in seed.yaml are unqiue too?

Reply:
Good idea, added.

Reply:
Enforcement of uniqueness of snap IDs will affect ability to install instances.